lib/stormpath-sdk/id_site_result.rb

```diff
@@ -0,0 +1,16 @@
+module Stormpath
+  class IdSiteResult
+    attr_accessor :account_href, :state, :status, :is_new_account
+
+    alias_method :is_new_account?, :is_new_account
```

Comment:
I don't think we need the "is_new_account?" alias, it should be enough with "new_account?" and "is_new_account". This can be removed.